lib/manager/actions/status.js

```diff
@@ -137,44 +138,73 @@ export async function fetchAppInfo () {
   }
 }
 
+/**
+ * Constructs modal content from an service period merge (principally used for
+ * MTC).
+ */
 function getMergeFeedModalContent (result: MergeFeedsResult): ModalContent {
+  const SEPARATOR = ': '
+  const {
+    errorCount,
+    failed,
+    failureReasons,
+    idConflicts,
+    mergeStrategy,
+    remappedIds,
+    remappedReferences,
+    skippedIds
+  } = result
+  const messages = getComponentMessages('MergeFeedsResult')
   const details = []
+  details.push(
+    [messages('strategyUsed'), messages(mergeStrategy)].join(SEPARATOR)
+  )
   // Do nothing or show merged feed modal? Feed version is be created
-  details.push('Remapped ID count: ' + result.remappedReferences)
-  if (Object.keys(result.remappedIds).length > 0) {
+  details.push([messages('remappedIdCount'), remappedReferences].join(SEPARATOR))
+  if (Object.keys(remappedIds).length > 0) {
     const remappedIdStrings = []
-    for (let key in result.remappedIds) {
+    for (const key in remappedIds) {
       // Modify key to remove feed name.
       const split = key.split(':')
       const tableAndId = split.splice(1, 1)
-      remappedIdStrings.push(`${tableAndId.join(':')} -> ${result.remappedIds[key]}`)
+      remappedIdStrings.push(`${tableAndId.join(':')} -> ${remappedIds[key]}`)
     }
-    details.push('Remapped IDs: ' + remappedIdStrings.join(', '))
+    details.push(
+      [messages('remappedIds'), remappedIdStrings.join(', ')].join(SEPARATOR)
+    )
   }
-  if (result.skippedIds.length > 0) {
+  if (skippedIds.length > 0) {
     const skippedRecordsForTables = {}
-    result.skippedIds.forEach(id => {
+    skippedIds.forEach(id => {
       const table = id.split(':')[0]
       // Increment count of skipped records for each value found per table.
       skippedRecordsForTables[table] = (skippedRecordsForTables[table] || 0) + 1
     })
     const skippedRecordsStrings = []
-    for (let key in skippedRecordsForTables) {
+    for (const key in skippedRecordsForTables) {
       skippedRecordsStrings.push(`${key} - ${skippedRecordsForTables[key]}`)
     }
-    details.push('Skipped records: ' + skippedRecordsStrings.join(', '))
+    details.push(
+      [messages('skipped'), skippedRecordsStrings.join(', ')].join(SEPARATOR)
```

Comment:
Could there be a template string used in the english.yml file instead?

Reply:
Improved in https://github.com/ibi-group/datatools-ui/pull/686/commits/1c1239097f50b08c9c2172d9de671f91ae96cc77 with a proper list.